change-notes/1.23/analysis-cpp.md

```diff
@@ -8,13 +8,15 @@ The following changes in version 1.23 affect C/C++ analysis in all applications.
 
 | **Query**                   | **Tags**  | **Purpose**                                                        |
 |-----------------------------|-----------|--------------------------------------------------------------------|
-| Query name (`query id`) | tags | Message. |
+| Hard-coded Japanese era start date (`cpp/japanese-era/exact-era-date`) | reliability, japanese-era | This query is a combination of two old queries that were identical in purpose but separate as an implementation detail.  This new query replaces Hard-coded Japanese era start date in call (`cpp/japanese-era/constructor-or-method-with-exact-era-date`) and Hard-coded Japanese era start date in struct (`cpp/japanese-era/struct-with-exact-era-date`). |
```

Comment:
We did not actually announce the two existing queries in the 1.22 change notes. See https://semmle.slack.com/archives/C7NS4H8HY/p1566982840001100.

@felicity-semmle, if you think this change note should be structured differently, let's discuss it on Slack.

Reply:
Would you prefer to just have a change note about the 'new' query, or no change note at all for this change?

Reply:
As those queries were provided by the community I think the current change note is appropriate. We don't want to fail to give credit to the original author by suggesting this is a brand new query.

Reply:
I'm fine with the change note as it is.